src/layman/layer/filesystem/gdal.py

```diff
@@ -288,6 +288,7 @@ def normalize_raster_file_async(input_path, crs_id, output_file):
         'gdalwarp',
         '-of', 'VRT',
         '-co', 'PROFILE=GeoTIFF',
+        '-co', 'BIGTIFF=YES',
```

Comment:
Does not make sense here as it's creating only VRT file. It should be inside compress_and_mask_raster_file_async where TIFF file is created.